src/ontology/bridge/uberon-bridge-to-fbbt.obo

```diff
@@ -1868,6 +1862,12 @@ property_value: IAO:0000589 "insect wing margin (drosophila)" xsd:string
 intersection_of: UBERON:6005378 ! insect wing margin
 intersection_of: part_of NCBITaxon:7227
 
+[Term]
+id: FBbt:00005380 ! 
+property_value: IAO:0000589 "insect pharynx (drosophila)" xsd:string
+intersection_of: UBERON:6005380 ! insect pharynx
+intersection_of: part_of NCBITaxon:7227
+
```

Comment:
This is the correct order, btw.

Reply:
yea i wonder how it could have been wrong before.

Reply:
Don't forget that currently the bridge files are generated by a Perl script; the OWL API rules about ordering do not apply. Frames are generated in the order the cross-references are encountered. Here, the order changed because the location of the cross-reference in the edit file changed with the obsoletion of the term with the bogus 8-digit ID and its replacement by a new term (whose frame is at a completely different location in the -edit file) with a correct ID.

Reply:
Ahh yes. Thank you!